### PR TITLE
Set secure defaults for DNS

### PR DESCRIPTION
**Pull request by TheEvilSkeleton**
Closes https://github.com/IngoZenz/personaldnsfilter/pull/103

### CODE REVIEW

**Comment by TheEvilSkeleton** [comment]
Does it automatically close once merged, like it should in GitLab?

**Comment by resynth1943** [comment]
Why did you create another Pr, Skel? 😛 
It looks good.

**Comment by resynth1943** [comment]
> Does it automatically close the referenced repository once merged, like it should in GitLab?

You can delete the branch from the UI, but deleting your forks is still quite a chore.

File: app/src/main/assets/dnsfilter.conf [hunk 1]
```diff
@@ -28,12 +26,12 @@ maxResolverCount = 100
 # This might be usefull in case applications ignore the VPN's DNS and just use an own DNS Server,
 # such as the Googles DNS Servers
 # default = "". Uncomment setting below for using this option.
-# As an example below the list of Google DNS servers for IPV4 and IPV6.
-# routeIPs = 8.8.8.8; 8.8.4.4; 2001:4860:4860::8888; 2001:4860:4860::8844
+# As an example below the list of secure and privacy respecting DNS servers for IPV4 and IPV6.
```
Comment (resynth1943) [inline app/src/main/assets/dnsfilter.conf:29]: Yes, but what DNS servers?

File: app/src/main/assets/dnsfilter.conf [hunk 2]
```diff
@@ -28,12 +26,12 @@ maxResolverCount = 100
 # This might be usefull in case applications ignore the VPN's DNS and just use an own DNS Server,
 # such as the Googles DNS Servers
```
Comment (resynth1943) [inline app/src/main/assets/dnsfilter.conf:27]: Should we remove this reference to Google?

**Comment by TheEvilSkeleton** [comment]
> Why did you create another Pr, Skel? stuck_out_tongue
> It looks good.

[Because he said that he wanted it in the Dev branch](https://github.com/IngoZenz/personaldnsfilter/pull/103#issuecomment-672145671), so I created a new PR to merge in the specific branch

**Comment by resynth1943** [comment]
Ahh, ok @TheEvilSkeleton ❤️

**Comment by CipherOps** [comment]
I'm not sure how to make it show the diffs here but this should fit your criteria.

File: app/src/main/assets/dnsfilter.conf (hunk 1 above)
Comment (resynth1943) [reply app/src/main/assets/dnsfilter.conf:43]: You posted that three times, by the way. I do think we get your point. 😛 

Are users aware that this module, if I'm understanding correctly, falls back to the router's DNS?

**Comment by IngoZenz** [comment]
1 more point -sorry ;-)
The samples starting in line 11 - I think they need to start with "#" in order to have them as comment ...

**Comment by CipherOps** [comment]
I've added examples, the list of DNS providers I have aren't examples but rather defaults. Do I need to move them all to the "fallback DNS" line?

**Comment by IngoZenz** [comment]
> I've added examples, the list of DNS providers I have aren't examples but rather defaults. Do I need to move them all to the "fallback DNS" line?

Exactly!
If you could now also revert the routeIPs comment, I think I could merge it ...

**Comment by CipherOps** [comment]
routeIPs now match the `0.0.0.0` and `::` and I've moved the list of Fallback DNS to the proper location.

**Pull request merged**